rust/extractor/src/translate/base.rs

```diff
@@ -743,9 +748,9 @@ impl<'a> Translator<'a> {
         }
         let ExpandResult {
             value: expanded, ..
-        } = semantics.expand_attr_macro(node)?;
-        self.emit_macro_expansion_parse_errors(node, &expanded);
-        let macro_items = ast::MacroItems::cast(expanded).or_else(|| {
+        } = self.semantics.and_then(|s| s.expand_attr_macro(node))?;
```

Comment:
Why `self.semantics.and_then` instead of using the local variable `semantics` defined above?

Reply:
that's a merge artifact, let me fix it